client/state/comments/schema.js

```diff
@@ -0,0 +1,83 @@
+export const commentsSchema = {
+	type: 'object',
+	patternProperties: {
+		'^\\d+-\\d+$': {
+			type: 'array',
+			items: {
+				type: 'object',
+				properties: {
+					ID: { type: 'integer' },
+					URL: { type: 'string' },
+					author: {
+						type: 'object',
+						properties: {
+							ID: { type: 'integer' },
+							URL: { type: 'string' },
+							avatar_URL: { type: 'string' },
+							email: { type: 'boolean' },
```

Comment:
Oh, the privacy settings! Makes sense.

I wasn't hung up on this, no worries. Just, I used a `PropTypes.string` that failed and I was curious to know why. :)